packages/java-parser/src/tokens.js

```diff
@@ -105,8 +106,11 @@ const UnarySuffixOperator = createToken({
   pattern: Lexer.NA
 });
 
-// TODO: align with Java Spec
-createToken({ name: "WhiteSpace", pattern: /\s+/, group: Lexer.SKIPPED });
+createToken({
```

Comment:
add a comment linking to: https://docs.oracle.com/javase/specs/jls/se11/html/jls-3.html#jls-3.6
above the token def.